docs/components/modeler/about-modeler.md

```diff
@@ -7,29 +7,29 @@ keywords: ["process mapping tool"]
 
-Camunda offers a few tools to design your diagrams and implement them:
+Camunda offers two tools to design and implement your diagrams:
 
 - [Web Modeler](./web-modeler/launch-cloud-modeler.md)
 - [Desktop Modeler](./desktop-modeler/index.md)
 
-Web Modeler and Desktop Modeler differ mainly in their environment. Web Modeler is part of Cloud Console and offers a seamless integration into Camunda Platform 8 to model BPMN. Desktop Modeler is a desktop application that can be installed and used locally, all while integrating your local development environment.
+Web Modeler and Desktop Modeler differ mainly in their environment. Web Modeler is part of Cloud Console and offers a seamless integration into Camunda Platform 8 to model BPMN. Desktop Modeler is a desktop application that can be installed and used locally to also design, view, and edit models, all while integrating your local development environment.
 
 :::note
-Interested in editing the underlying XML of BPMN and DMN diagrams directly? Be sure to utilize Desktop Modeler, as it features an XML editor tab. The XML editor comes with its own history (undo or redo), search and replace functionality, and XML syntax highlighting.
+Interested in editing the underlying XML of BPMN and [Decision Model and Notation (DMN)](./dmn/dmn.md) diagrams directly? Be sure to utilize Desktop Modeler, as it features an XML editor tab. The XML editor comes with its own history (undo or redo), search and replace functionality, and XML syntax highlighting.
 :::
 
 In this guide, we'll demonstrate modeling BPMN diagrams using both Web Modeler and Desktop Modeler.
 
 ## Next steps
 
-- [Modeling BPMN](/guides/automating-a-process-using-bpmn.md) - Learn how to quickly model an automated process using BPMN.
-- [Camunda Forms](/guides/utilizing-forms.md) - Allows you to easily design and configure forms. Once configured, they can be connected to a user task or start event to implement a task form in your application.
+- [Modeling BPMN](/guides/automating-a-process-using-bpmn.md) - Learn how to model an automated process in this tutorial using Web Modeler with Camunda Platform 8 SaaS.
+- [Camunda Forms](/guides/utilizing-forms.md) - Allows you to design and configure forms. Once configured, they can be connected to a user task or start event to implement a task form in your application.
 - [DMN](./dmn/dmn.md) - In DMN, decisions can be modeled and executed using the same language. Business analysts can model the rules that lead to a decision in easy to read tables, and those tables can be executed directly by a decision engine (like Camunda).
 - [Out-of-the-box Connectors](/guides/configuring-out-of-the-box-connector.md) - Out-of-the-box Connectors provide prebuilt connectivity with a number of outside systems. Connectors are modular by nature, reusable, and accelerate automation across those systems.
```

Comment:
Proposal:

* Remove this and/or introduce a section called "discover/use connectors" => It is powerful, and we want people to know them.